db/migrate/20161128193435_add_researcher_quote_to_project_contents.rb

```diff
@@ -0,0 +1,7 @@
+class AddResearcherQuoteToProjectContents < ActiveRecord::Migration
+  def change
+    add_column :project_contents, :researcher_quote, :text
```

Comment:
adding the column with a default will backfill, https://www.postgresql.org/docs/9.3/static/ddl-alter.html. However it'll lock the table to add that col and depending on table size it might be a while. If it's too long for normal ops then we'd have to add without default, backfill with rake task and then add default col val.